visual-studio-vstsbuildagent-vm/InstallVSTSAgent.ps1

```diff
@@ -81,7 +83,88 @@ Write-Verbose "Configuring agent" -Verbose
 # Set the current directory to the agent dedicated one previously created.
 Push-Location -Path $agentInstallationPath
 
-.\config.cmd --unattended --url $serverUrl --auth PAT --token $PersonalAccessToken --pool $PoolName --agent $AgentName --runasservice
+if ($runAsAutoLogon -ieq "true")
+{
+  # Create a PS session for the user to trigger the creation of the registry entries required for autologon
+  $computerName = "localhost"
+  $password�=�ConvertTo-SecureString�$vmAdminPassword -AsPlainText�-Force
```

Comment:
>� [](start = 11, length = 1)

you have invalid characters.. please fix it..

Reply:
hmmm, missed these while cleaning up the rest. Plus stopped looking as it worked find. Will clean all of them up. There were a few in the original script as well and they hadn't been cauing any issues, I guess it's only in certain places they bomb

Reply:
They don't show up in notepad++ under ANSI enoding, only codeflow seems to show them